python/docs/conf.py

```diff
@@ -48,16 +48,16 @@
 
 # General information about the project.
 project = u'PySpark'
-copyright = u'2014, Author'
+copyright = u''
```

Comment:
I did not figure out the right answer here, so leave it as blank. We did not have this kind of information in the Java/Scala API docs.